### ISSUE TITLE
2025-03-06, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#103]
# Today's Arxiv Papers

(1) [The MASK Benchmark: Disentangling Honesty From Accuracy in AI Systems](http://arxiv.org/abs/2503.03750v1), 2025-03-05

(2) [Process-based Self-Rewarding Language Models](http://arxiv.org/abs/2503.03746v1), 2025-03-05

(3) [Towards Understanding Distilled Reasoning Models: A Representational Approach](http://arxiv.org/abs/2503.03730v1), 2025-03-05

(4) [Improving LLM Safety Alignment with Dual-Objective Optimization](http://arxiv.org/abs/2503.03710v1), 2025-03-05

(5) [Effective LLM Knowledge Learning via Model Generalization](http://arxiv.org/abs/2503.03705v1), 2025-03-05

(6) [A Practical Memory Injection Attack against LLM Agents](http://arxiv.org/abs/2503.03704v1), 2025-03-05

(7) [Developing and Utilizing a Large-Scale Cantonese Dataset for Multi-Tasking in Large Language Models](http://arxiv.org/abs/2503.03702v1), 2025-03-05

(8) [Addressing Overprescribing Challenges: Fine-Tuning Large Language Models for Medication Recommendation Tasks](http://arxiv.org/abs/2503.03687v1), 2025-03-05

(9) [MAS-GPT: Training LLMs to Build LLM-based Multi-Agent Systems](http://arxiv.org/abs/2503.03686v1), 2025-03-05

(10) [Attentive Reasoning Queries: A Systematic Method for Optimizing Instruction-Following in Large Language Models](http://arxiv.org/abs/2503.03669v1), 2025-03-05

(11) [Analogical Reasoning Inside Large Language Models: Concept Vectors and the Limits of Abstraction](http://arxiv.org/abs/2503.03666v1), 2025-03-05

(12) [Improving Neutral Point of View Text Generation through Parameter-Efficient Reinforcement Learning and a Small-Scale High-Quality Dataset](http://arxiv.org/abs/2503.03654v1), 2025-03-05

(13) [Token-Level Privacy in Large Language Models](http://arxiv.org/abs/2503.03652v1), 2025-03-05

(14) [Psy-Copilot: Visual Chain of Thought for Counseling](http://arxiv.org/abs/2503.03645v1), 2025-03-05

(15) [Large language models in finance: estimating financial sentiment for stock prediction](http://arxiv.org/abs/2503.03612v1), 2025-03-05

(16) [Enhancing the Accuracy and Comprehensibility in Architectural Tactics Detection via Small Model-Augmented Prompt Engineering](http://arxiv.org/abs/2503.03609v1), 2025-03-05

(17) [Psy-Insight: Explainable Multi-turn Bilingual Dataset for Mental Health Counseling](http://arxiv.org/abs/2503.03607v1), 2025-03-05

(18) [Feature-Level Insights into Artificial Text Detection with Sparse Autoencoders](http://arxiv.org/abs/2503.03601v1), 2025-03-05

(19) [Small but Mighty: Enhancing Time Series Forecasting with Lightweight LLMs](http://arxiv.org/abs/2503.03594v1), 2025-03-05

(20) [English K_Quantization of LLMs Does Not Disproportionately Diminish Multilingual Performance](http://arxiv.org/abs/2503.03592v1), 2025-03-05

(21) [PowerAttention: Exponentially Scaling of Receptive Fields for Effective Sparse Attention](http://arxiv.org/abs/2503.03588v1), 2025-03-05

(22) ["You don't need a university degree to comprehend data protection this way": LLM-Powered Interactive Privacy Policy Assessment](http://arxiv.org/abs/2503.03587v1), 2025-03-05

(23) [Benchmarking LLMs and LLM-based Agents in Practical Vulnerability Detection for Code Repositories](http://arxiv.org/abs/2503.03586v1), 2025-03-05

(24) [Afford-X: Generalizable and Slim Affordance Reasoning for Task-oriented Manipulation](http://arxiv.org/abs/2503.03556v1), 2025-03-05

(25) [Parallelized Planning-Acting for Efficient LLM-based Multi-Agent Systems](http://arxiv.org/abs/2503.03505v1), 2025-03-05

(26) [Collaborative Expert LLMs Guided Multi-Objective Molecular Optimization](http://arxiv.org/abs/2503.03503v1), 2025-03-05

(27) [CURVALID: Geometrically-guided Adversarial Prompt Detection](http://arxiv.org/abs/2503.03502v1), 2025-03-05

(28) [Open-Source Large Language Models as Multilingual Crowdworkers: Synthesizing Open-Domain Dialogues in Several Languages With No Examples in Targets and No Machine Translation](http://arxiv.org/abs/2503.03462v1), 2025-03-05

(29) [Visualising Policy-Reward Interplay to Inform Zeroth-Order Preference Optimisation of Large Language Models](http://arxiv.org/abs/2503.03460v1), 2025-03-05

(30) [Unified Mind Model: Reimagining Autonomous Agents in the LLM Era](http://arxiv.org/abs/2503.03459v1), 2025-03-05

(31) [Taxation Perspectives from Large Language Models: A Case Study on Additional Tax Penalties](http://arxiv.org/abs/2503.03444v1), 2025-03-05

(32) [RASD: Retrieval-Augmented Speculative Decoding](http://arxiv.org/abs/2503.03434v1), 2025-03-05

(33) [When Claims Evolve: Evaluating and Enhancing the Robustness of Embedding Models Against Misinformation Edits](http://arxiv.org/abs/2503.03417v1), 2025-03-05

(34) [The Serendipity of Claude AI: Case of the 13 Low-Resource National Languages of Mali](http://arxiv.org/abs/2503.03380v1), 2025-03-05

(35) [Leveraging Large Language Models to Develop Heuristics for Emerging Optimization Problems](http://arxiv.org/abs/2503.03350v1), 2025-03-05

(36) [EnigmaToM: Improve LLMs' Theory-of-Mind Reasoning Capabilities with Neural Knowledge Base of Entity States](http://arxiv.org/abs/2503.03340v1), 2025-03-05

(37) [iNews: A Multimodal Dataset for Modeling Personalized Affective Responses to News](http://arxiv.org/abs/2503.03335v1), 2025-03-05

(38) [LLM as GNN: Graph Vocabulary Learning for Text-Attributed Graph Foundation Models](http://arxiv.org/abs/2503.03313v1), 2025-03-05

(39) [SEOE: A Scalable and Reliable Semantic Evaluation Framework for Open Domain Event Detection](http://arxiv.org/abs/2503.03303v1), 2025-03-05

(40) [LexGenie: Automated Generation of Structured Reports for European Court of Human Rights Case Law](http://arxiv.org/abs/2503.03266v1), 2025-03-05

(41) [A 262 TOPS Hyperdimensional Photonic AI Accelerator powered by a Si3N4 microcomb laser](http://arxiv.org/abs/2503.03263v1), 2025-03-05

(42) [Can Frontier LLMs Replace Annotators in Biomedical Text Mining? Analyzing Challenges and Exploring Solutions](http://arxiv.org/abs/2503.03261v1), 2025-03-05

(43) [Exploring the Potential of Large Language Models as Predictors in Dynamic Text-Attributed Graphs](http://arxiv.org/abs/2503.03258v1), 2025-03-05

(44) [PAIR: A Novel Large Language Model-Guided Selection Strategy for Evolutionary Algorithms](http://arxiv.org/abs/2503.03239v1), 2025-03-05

(45) [FANS -- Formal Answer Selection for Natural Language Math Reasoning Using Lean4](http://arxiv.org/abs/2503.03238v1), 2025-03-05

(46) [Targeted Distillation for Sentiment Analysis](http://arxiv.org/abs/2503.03225v1), 2025-03-05

(47) [COSINT-Agent: A Knowledge-Driven Multimodal Agent for Chinese Open Source Intelligence](http://arxiv.org/abs/2503.03215v1), 2025-03-05

(48) [PolyVer: A Compositional Approach for Polyglot System Modeling and Verification](http://arxiv.org/abs/2503.03207v1), 2025-03-05

(49) [MA-LoT: Multi-Agent Lean-based Long Chain-of-Thought Reasoning enhances Formal Theorem Proving](http://arxiv.org/abs/2503.03205v1), 2025-03-05

(50) [Towards Robust Universal Information Extraction: Benchmark, Evaluation, and Solution](http://arxiv.org/abs/2503.03201v1), 2025-03-05

(51) [Structured Outputs Enable General-Purpose LLMs to be Medical Experts](http://arxiv.org/abs/2503.03194v1), 2025-03-05

(52) [Enhancing Memory Efficiency in Large Language Model Training Through Chronos-aware Pipeline Parallelism](http://arxiv.org/abs/2503.03182v1), 2025-03-05

(53) [Enhancing Cybersecurity in Critical Infrastructure with LLM-Assisted Explainable IoT Systems](http://arxiv.org/abs/2503.03180v1), 2025-03-05

(54) [AttackSeqBench: Benchmarking Large Language Models' Understanding of Sequential Patterns in Cyber Attacks](http://arxiv.org/abs/2503.03170v1), 2025-03-05

(55) [Dango: A Mixed-Initiative Data Wrangling System using Large Language Model](http://arxiv.org/abs/2503.03154v1), 2025-03-05

(56) [DSVD: Dynamic Self-Verify Decoding for Faithful Generation in Large Language Models](http://arxiv.org/abs/2503.03149v1), 2025-03-05

(57) [PriFFT: Privacy-preserving Federated Fine-tuning of Large Language Models via Function Secret Sharing](http://arxiv.org/abs/2503.03146v1), 2025-03-05

(58) [Bridging Molecular Graphs and Large Language Models](http://arxiv.org/abs/2503.03135v1), 2025-03-05

(59) [Towards Understanding Multi-Round Large Language Model Reasoning: Approximability, Learnability and Generalizability](http://arxiv.org/abs/2503.03128v1), 2025-03-05

(60) [The Devil Is in the Details: Tackling Unimodal Spurious Correlations for Generalizable Multimodal Reward Models](http://arxiv.org/abs/2503.03122v1), 2025-03-05

(61) [PromAssistant: Leveraging Large Language Models for Text-to-PromQL](http://arxiv.org/abs/2503.03114v1), 2025-03-05

(62) [SoK: Knowledge is All You Need: Last Mile Delivery for Automated Provenance-based Intrusion Detection with LLMs](http://arxiv.org/abs/2503.03108v1), 2025-03-05

(63) [Monitoring Decoding: Mitigating Hallucination via Evaluating the Factuality of Partial Response during Generation](http://arxiv.org/abs/2503.03106v1), 2025-03-05

(64) [BEVDriver: Leveraging BEV Maps in LLMs for Robust Closed-Loop Driving](http://arxiv.org/abs/2503.03074v1), 2025-03-05

(65) [Improving LLM-as-a-Judge Inference with the Judgment Distribution](http://arxiv.org/abs/2503.03064v1), 2025-03-04

(66) [SAGE: Steering and Refining Dialog Generation with State-Action Augmentation](http://arxiv.org/abs/2503.03040v1), 2025-03-04

(67) [LLM Misalignment via Adversarial RLHF Platforms](http://arxiv.org/abs/2503.03039v1), 2025-03-04

(68) [SAFE: A Sparse Autoencoder-Based Framework for Robust Query Enrichment and Hallucination Mitigation in LLMs](http://arxiv.org/abs/2503.03032v1), 2025-03-04

(69) [Zero-Shot Multi-Label Classification of Bangla Documents: Large Decoders Vs. Classic Encoders](http://arxiv.org/abs/2503.02993v1), 2025-03-04

(70) [Effectively Steer LLM To Follow Preference via Building Confident Directions](http://arxiv.org/abs/2503.02989v1), 2025-03-04

(71) [Teaching AI to Handle Exceptions: Supervised Fine-Tuning with Human-Aligned Judgment](http://arxiv.org/abs/2503.02976v1), 2025-03-04

(72) [LINGOLY-TOO: Disentangling Memorisation from Reasoning with Linguistic Templatisation and Orthographic Obfuscation](http://arxiv.org/abs/2503.02972v1), 2025-03-04

(73) [Multilingual Relative Clause Attachment Ambiguity Resolution in Large Language Models](http://arxiv.org/abs/2503.02971v1), 2025-03-04

(74) [InfiniSST: Simultaneous Translation of Unbounded Speech with Large Language Model](http://arxiv.org/abs/2503.02969v1), 2025-03-04

(75) [KodCode: A Diverse, Challenging, and Verifiable Synthetic Dataset for Coding](http://arxiv.org/abs/2503.02951v1), 2025-03-04

(76) [ExpertGenQA: Open-ended QA generation in Specialized Domains](http://arxiv.org/abs/2503.02948v1), 2025-03-04

(77) [Wikipedia in the Era of LLMs: Evolution and Risks](http://arxiv.org/abs/2503.02879v1), 2025-03-04

(78) [Language Models can Self-Improve at State-Value Estimation for Better Search](http://arxiv.org/abs/2503.02878v1), 2025-03-04

(79) [The First Few Tokens Are All You Need: An Efficient and Effective Unsupervised Prefix Fine-Tuning Method for Reasoning Models](http://dx.doi.org/10.13140/RG.2.2.33772.07043), 2025-03-04

(80) [Prompting Generative AI with Interaction-Augmented Instructions](http://arxiv.org/abs/2503.02874v1), 2025-03-04

(81) [Optimizing open-domain question answering with graph-based retrieval augmented generation](http://arxiv.org/abs/2503.02922v1), 2025-03-04

(82) [FairSense-AI: Responsible AI Meets Sustainability](http://arxiv.org/abs/2503.02865v2), 2025-03-04

(83) [Calibrating LLM Confidence with Semantic Steering: A Multi-Prompt Aggregation Framework](http://arxiv.org/abs/2503.02863v1), 2025-03-04

(84) [Privacy and Accuracy-Aware AI/ML Model Deduplication](http://arxiv.org/abs/2503.02862v1), 2025-03-04

(85) [Shakespearean Sparks: The Dance of Hallucination and Creativity in LLMs' Decoding Layers](http://arxiv.org/abs/2503.02851v1), 2025-03-04

(86) [Mask-DPO: Generalizable Fine-grained Factuality Alignment of LLMs](http://arxiv.org/abs/2503.02846v1), 2025-03-04

(87) [AlignDistil: Token-Level Language Model Alignment as Adaptive Policy Distillation](http://arxiv.org/abs/2503.02832v1), 2025-03-04

(88) [Q-Filters: Leveraging QK Geometry for Efficient KV Cache Compression](http://arxiv.org/abs/2503.02812v1), 2025-03-04

(89) [RAAD-LLM: Adaptive Anomaly Detection Using LLMs and RAG Integration](http://arxiv.org/abs/2503.02800v1), 2025-03-04

(90) [IterPref: Focal Preference Learning for Code Generation via Iterative Debugging](http://arxiv.org/abs/2503.02783v1), 2025-03-04

(91) [Multimodal AI predicts clinical outcomes of drug combinations from preclinical data](http://arxiv.org/abs/2503.02781v1), 2025-03-04

(92) [Implicit Bias in LLMs: A Survey](http://arxiv.org/abs/2503.02776v1), 2025-03-04

(93) [InSerter: Speech Instruction Following with Unsupervised Interleaved Pre-training](http://arxiv.org/abs/2503.02769v1), 2025-03-04

(94) [From Metaphor to Mechanism: How LLMs Decode Traditional Chinese Medicine Symbolic Language for Modern Clinical Relevance](http://arxiv.org/abs/2503.02760v1), 2025-03-04

(95) [BatchGEMBA: Token-Efficient Machine Translation Evaluation with Batched Prompting and Prompt Compression](http://arxiv.org/abs/2503.02756v1), 2025-03-04

(96) [Large Language Models for Multilingual Previously Fact-Checked Claim Detection](http://arxiv.org/abs/2503.02737v1), 2025-03-04

(97) [Evaluating Knowledge Generation and Self-Refinement Strategies for LLM-based Column Type Annotation](http://arxiv.org/abs/2503.02718v1), 2025-03-04

(98) [RedChronos: A Large Language Model-Based Log Analysis System for Insider Threat Detection in Enterprises](http://arxiv.org/abs/2503.02702v1), 2025-03-04

(99) [MindBridge: Scalable and Cross-Model Knowledge Editing via Memory-Augmented Modality](http://arxiv.org/abs/2503.02701v1), 2025-03-04

(100) [FlowPlan: Zero-Shot Task Planning with LLM Flow Engineering for Robotic Instruction Following](http://arxiv.org/abs/2503.02698v1), 2025-03-04
